app/javascript/packs/application.js

```diff
@@ -1,5 +1,11 @@
 import loadPolyfills from '../mastodon/load_polyfills';
 
+// import default stylesheet with variables
+require('font-awesome/css/font-awesome.css');
+require('mastodon-application-style');
+
+require.context('../images/', true);
+
```

Comment:
`application.js` isn't loaded by multi-frontend stuff, that's `frontends/mastodon.js`. This file might not even be used anymore?? (If it is, it's just to display the timelines on the `/about` page.

We should keep the file around though for upstream compatibility